frontend/src/scenes/session-recordings/playlist/SessionRecordingPreview.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import { Meta } from '@storybook/react'
+
+import { PropertyIcons, PropertyIconsProps } from './SessionRecordingPreview'
+
+const meta: Meta<typeof PropertyIcons> = {
+    title: 'Replay/Components/PropertyIcons',
+    component: PropertyIcons,
+    parameters: {
+        layout: 'fullscreen',
+        viewMode: 'story',
+        mockDate: '2023-01-31 12:00:00',
+    },
+}
+export default meta
+
+const Template = (args: PropertyIconsProps): JSX.Element => {
+    return (
+        <div className={'border rounded px-4 py-2 w-fit'}>
+            <PropertyIcons {...args} />
+        </div>
+    )
+}
+
+export const WebRecording = (): JSX.Element => {
+    return (
+        <Template
+            iconClassnames={'wat'}
+            loading={false}
+            onPropertyClick={() => {}}
+            recordingProperties={[
+                { value: 'Mac OS X', property: '$os', tooltipValue: 'Mac OS X' },
```

Comment:
```suggestion
                { value: 'macOS', property: '$os', tooltipValue: 'macOS' },
```

Reply:
the web sdk sends the literal string 'Mac OS X' though :)

Reply:
Time to fix it? :)
https://setapp.com/how-to/full-list-of-all-macos-versions

> Is there any difference between Mac OS X and macOS?
> No, they are essentially the same thing — just named differently. In fact, three terms were used at different times with reference to Apple’s operating system: Mac OS X, OS X, and macOS. Mac OS X was the official naming through version 10.7, from 2001 to 2011. In the next four years, the OS X names were used. Finally, Apple shifted to "macOS" with the release of macOS High Sierra in 2016. The latter helped standardize the naming of Apple’s operating systems — macOS, iOS, tvOS, iPadOS, etc.

Reply:
it'll be a JS SDK change and we need to support existing... but added an additional row and a comment so we're ready for the fix in the web sdk

Reply:
🤔 need to think about existing insights that query for `Mac OS X` 🙈

Reply:
If it's gonna break things and/or is hard to fix and keep back compatibility, let's keep it as it is (`Mac OS X`), but let's add a comment and/or create a new issue for fixing it in the future, the current form isn't used for years but it's not wrong, it's just the former name.

Reply:
yeah, let's start a slower process to see if we do/should/can fix. It's an interesting question because it won't be the last time we run across something similar